core/src/main/scala/spinal/core/internals/Phase.scala

```diff
@@ -1420,6 +1422,14 @@ class PhaseDeviceDefault extends PhaseDeviceHandler{
       mem.addAttribute("ramstyle", "no_rw_check")
     }
   }
+  override def onCrossClockMaxDelay(config: SpinalConfig, sources: Iterable[BaseType], target: BaseType) = {
+    if(config.device.vendor == Device.ALTERA.vendor){
```

Comment:
I would say, that it could also be enabled for config.device.isVendorDefault. that would make things safer for everyone ^^ unless it can break some flow ?

Reply:
Else i think this can be merged, i may touch things a bit.

Reply:
Yes good point. I do not see this clearly altera marked attribute confusing anything else.
I am good with merging this as well.

Reply:
Ok ^^ 

I should be able to merge monday :)